modules/primer-breadcrumb/lib/breadcrumb.scss

```diff
@@ -12,6 +11,7 @@
   }
 }
 
+:last-child .breadcrumb-item,
```

Comment:
This change removes the forward slash off of every breadcrumb, unsure what this change was trying to achieve but we don't want to merge it like this.